ff/src/fields/models/cubic_extension.rs

```diff
@@ -372,8 +372,11 @@ impl<P: CubicExtParameters> FromBytes for CubicExtField<P> {
 impl<P: CubicExtParameters> Neg for CubicExtField<P> {
     type Output = Self;
     #[inline]
-    fn neg(self) -> Self {
-        Self::new(self.c0.neg(), self.c1.neg(), self.c2.neg())
+    fn neg(mut self) -> Self {
```

Comment:
I still don't understand why this change is useful, does it show up in benchmarks or is it aesthetic?

Reply:
it modifies the elements in place, instead of making a new one. I don't think it shows up in benchmarks, at least not enough to affect things greatly

Reply:
Sorry, I didn't mean the `mut` bit, I mean changing the `self.c0.neg` to `self.c0 = - self.c0`. I commented the wrong line 😅

Reply:
Ah I understand this now though, neg is not in place,